symphony-ws/src/main/java/se/havochvatten/symphony/service/ReportService.java

```diff
@@ -420,4 +415,290 @@ static void writeComparisonRows(Locale locale, SymphonyBandDto[] meta, int[] fea
+                        ODF_TITLE_ROWS + e);
+                    nextCell.setDoubleValue(layersToList.result[p][e]);
+                    nextCell.getOdfElement().setStyleName("value");
+                }
+            }
+
+            nextCell = nextSheet.getCellByPosition(
+                2 + layersToList.pressures.length,
+                ODF_TITLE_ROWS - 1);
+            nextCell.setStringValue("TOTAL");
+            nextCell.getOdfElement().setStyleName("totalHE");
+
+            nextCell = nextSheet.getCellByPosition(
+                1,
+                ODF_TITLE_ROWS + layersToList.ecosystems.length);
+            nextCell.setStringValue("TOTAL");
+            nextCell.getOdfElement().setStyleName("totalHP");
+
+            for (e = 0; e < layersToList.ecosystems.length; e++) {
+                nextCell = nextSheet.getCellByPosition(
+                    2 + layersToList.pressures.length,
+                    ODF_TITLE_ROWS + e);
+                nextCell.setDoubleValue(cmp.totalPerEcosystem.get(cmp.includedEcosystems[layersToList.ecosystems[e]]));
+                nextCell.getOdfElement().setStyleName("totalE");
+            }
+
+            for (p = 0; p < layersToList.pressures.length; p++) {
+                nextCell = nextSheet.getCellByPosition(
+                    2 + p,
+                    ODF_TITLE_ROWS + layersToList.ecosystems.length);
+                nextCell.setDoubleValue(cmp.totalPerPressure.get(cmp.includedPressures[layersToList.pressures[p]]));
+                nextCell.getOdfElement().setStyleName("totalP");
+            }
+
+            nextCell = nextSheet.getCellByPosition(
+                2 + layersToList.pressures.length,
+                ODF_TITLE_ROWS + layersToList.ecosystems.length);
+            nextCell.setDoubleValue(cmp.cumulativeTotal);
+            nextCell.getOdfElement().setStyleName("totalC");
+
+            // ----- Total sheet -----
+
+            nextCell = totalSheet.getCellByPosition(1, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL);
+            nextCell.setStringValue(cmp.calculationName);
+            nextCell.getOdfElement().setStyleName("calcName");
+            totalSheet.getRowByIndex(cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 2).setHeight(2, false);
+            totalSheet.getRowByIndex(cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 5).setHeight(2, false);
+
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL);
+            nextCell.setStringValue("Pressure");
+            nextCell.getOdfElement().setStyleName("pressureHeader");
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 1);
+            nextCell.setStringValue("TOTAL");
+            nextCell.getOdfElement().setStyleName("totalHP");
+
+            nextCell = totalSheet.getCellByPosition(2, cmpIndex * 7 + ODF_TITLE_ROWS_TOTAL + 3);
+            nextCell.setStringValue("Ecosystem");
+            nextCell.getOdfElement().setStyleName("pressureHeader");
```

Comment:
Is this correct?

Reply:
Yes. it is correct, but I agree that it's a bit hard on the eyes :confounded:
I'll amend the PR to set 7 as a static constant (like some other arbitrary numbers).

Reply:
To clarify, I think the number 3 is transparent enough here - as an offset from the directly preceding row. 
Maybe a comment ti this effect is appropriate.

Reply:
I ment the style name here as well :)

Reply:
Please refer to the explanation in the other thread!